backend/lib/calculate/adequacy.py

```diff
@@ -149,7 +128,7 @@ def _get_addresses_to_check_by_service_area(
 @timed
 def calculate_adequacies(
     service_area_ids,
-    provider_ids,
+    addresses,
```

Comment:
Ditto about these not really being addresses anymore.

Reply:
Done